starter-repos/app-template/skaffold.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: skaffold/v2beta7
+kind: Config
+metadata:
+  name: app-template
+build:
+  artifacts:
+  - image: gcr.io/jingfangliu-194020/app:v1
```

Comment:
How about we change it to `image: app` and add a comment to let user customize it? If they don't customize it, it should also work.

Reply:
+1